app/views/taxons/_organisation_logos_and_list.html.erb

```diff
@@ -2,7 +2,7 @@
   <ol>
     <% organisations_with_logos.each do |promoted_organisation| %>
       <li class="taxon-page__organisation">
-        <%= render 'govuk_component/organisation_logo', {
+        <%= render 'govuk_publishing_components/components/organisation_logo', {
```

Comment:
This should perhaps be part of the first commit?

Reply:
Not sure I agree, not sure I have time to address anyway.

Reply:
This change is related to the taxons view rather than organisation pages, which is why I don't think it belongs in this commit. Happy to help if you decide you want to split this out.

Reply:
Done.